OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
> Do you have any other model file in mind ?

As far as I can tell, none of the models currently in the repo contain a muscle whose `isDisabled` property is set to `true`.

> The `ASSERT` is checking the property of the model read from the old file and new file.

I thought the `updateFromXMLNode()` code will be executed when the model is loaded (L1683), so the test is comparing a v30508 model to the same v30508 model (after serialization/deserialization). Not saying that's a bad thing to test, but it seems like that type of check should be done in a more general way for all properties (if it isn't being tested already).